source/js/components/Form/Form.tsx

```diff
@@ -0,0 +1,13 @@
+import { FormEventHandler } from 'react';
```

Comment:
Missing tests for for, but that's perhaps fine to begin with

Reply:
Yes me too. We have to add more tests over time, it felt out of scope to add tests for every component for this task.